style.css

```diff
@@ -530,358 +538,306 @@ input[type="text"] {
-      margin-left: -107px;
-    }
-
-    .form-body {
-      justify-content: center;
-      margin: 1px;
-    }
-
-    .ul_form {
-      display: flex;
-      flex-direction: column;
-      align-items: center;
-      padding: 1px;
-    }
-
-    input {
-      height: 48px;
-      width: 447px;
-      background: #fff;
-      border: 1px solid #cfd8dc;
-      border-radius: 8px;
-      margin-bottom: 18px;
-    }
-
-    textarea {
-      height: 170px;
-      width: 447px;
-      bottom: 0;
-      background: #fff;
-      border: 1px solid #cfd8dc;
-      border-radius: 8px;
-      margin-top: 18px;
-      font-family: 'Poppins', sans-serif;
-      font-style: normal;
-    }
-
-    .form p {
-      width: 400px;
-    }
-
-    .form-header {
-      padding-top: 80px;
-      text-align: center;
-      padding-left: 24px;
-      padding-right: 24px;
-      flex-direction: column;
-      align-items: center;
-    }
-
-    li .for-web img {
-      width: 80%;
-    }
-
-    li .for-web p {
-      padding-top: 0;
-    }
-
-    .form textarea {
-      height: 170px;
-      width: 447px;
-      bottom: 0;
-      background: #fff;
-      border: 1px solid #cfd8dc;
-      border-radius: 8px;
-      margin-top: 18px;
-      font-family: 'Poppins', sans-serif;
-      font-style: normal;
-    }
-
-    .form .form-header {
-      display: flex;
-      padding-top: 80px;
-      text-align: center;
-      padding-left: 24px;
-      padding-right: 24px;
-      flex-direction: column;
-      align-items: center;
-    }
-
-    .main-header .logo {
-      transition-property: transform;
-      transition-duration: 1.5s;
-    }
-
-    .logo:hover {
-      transform: scale(1.2) rotate(360deg);
-    }
-
-    button {
-      transition-property: background-color, color, transform;
-      transition-duration: 0.5s;
-    }
-
-    button:hover {
-      transform: scale(1.1);
-      background-color: #6070ff;
-      color: #fff;
-    }
-
-    .about-para {
-      width: 459px;
-      height: 102px;
-      font-family: 'Poppins', sans-serif;
-      font-size: 16px;
-      font-weight: 400;
-      line-height: 20px;
-      letter-spacing: 0;
-      text-align: left;
-      margin-left: 107px;
-    }
-
-    .myself {
-      margin-left: 148px;
-    }
-
-    .about-sub-head {
-      margin-left: 148px;
-    }
-
-    #image-header img {
-      padding-left: 95%;
-    }
+  }
+
+  .logo {
+    margin-right: auto;
+    margin-left: 155px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .desk {
+    display: inline;
+    margin-left: -85px;
+    margin-right: 117px;
+    text-decoration: none;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #344563;
+  }
+
+  .part1 {
+    background: url(images/5.png);
+    background-position: cover;
+    background-repeat: no-repeat;
+    background-size: 100%;
+  }
+
```

Comment:
[OPTIONAL]
- [ ] Kindly add some space between the `.part1` and `.container` sections and remove the gaps at the left and right side of your website to stick to the Figma Design as requested in the project requirement (see image).

### Your Design
![Screenshot from 2023-04-06 10-05-27](https://user-images.githubusercontent.com/91142432/230356122-5d072b7d-c4c3-4ae9-9343-7781045c362e.png)

### Figma's Design
![Screenshot from 2023-04-06 10-52-27](https://user-images.githubusercontent.com/91142432/230356493-3c2292bd-c5ea-4988-8a01-ed54763af1f4.png)

This is a requirement of the project but the first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change